### ISSUE TITLE
[cli] add --version option?

### ISSUE THREAD

**Issue by severo** [hyparam/hyperparam-cli#57]
It should do the following:

```
$ npm i -g hyperparam
$ hyperparam --version
2.0.0
```

Also: document the option in `--help`

**Comment by severo** [comment]
#120 did not fix the issue, after all

```
$ npm i -g hyperparam
$ hyperparam --help   
Usage:
  hyperparam [path]         start hyperparam webapp. "path" is a directory or a URL.
                            defaults to the current directory.
  hyperparam chat           start chat client
  
  hyperparam -h, --help,    give this help list
  hyperparam -v, --version  print program version
$ hyperparam --version
undefined
```

**Comment by severo** [comment]
fixed by #122
